README.md

```diff
@@ -1 +1,69 @@
-# kanban-board
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Movies Database
+
+> A webapp to find information about a collection of movies/shows. You can like your favorite shows, leave a commaent and make reservations for screening.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript ES6
+- Webpack
+- TvMaze API
+- Involvement API
+- Jest
+
+## Live Demo
+
+[Live Demo Link](https://drive.google.com/file/d/1GuBA_O6cSGohCIswKyfx4E8-lEnLcqyX/view?usp=sharing)
+
+## Video Presentation
+
+[Video Presentation Link](https://drive.google.com/file/d/1GuBA_O6cSGohCIswKyfx4E8-lEnLcqyX/view?usp=sharing)
```

Comment:
- [ ] Good job team 👌🏼👏🏼🙌🏼. As a requirement video presentation should be long max 5 mins. Please rerecord another shorter video 📽. The app is working correctly 😃